## Importações de Bibliotecas

In [56]:
# Biblioteca para analise e importações dos arquivos 
import pandas as pd

### Fazendo conexão com o DRIVE. 

In [ ]:
#teste teste
https://github.com/FtcThiago/Analise_de_dados/blob/main/Analise_de_dados_Python/Projeto_censo_educacao_superior.ipynb
origin  https://github.com/FtcThiago/Analise_de_dados.git (fetch)
origin  https://github.com/FtcThiago/Analise_de_dados.git (push)

### 1. Definição do Problema

# Taxas de Desistência no Ensino Superior do Distrito Federal
Análise das Taxas de Desistência no Ensino Superior do Distrito Federal: Um Estudo nas Dez Instituições Mais Bem Avaliadas pelo MEC em 2021

## Objetivo da Análise
Este estudo tem como objetivo analisar e compreender os fatores associados às taxas de desistência em cursos de graduação no Distrito Federal. A investigação será conduzida com base em dados públicos extraídos de plataformas governamentais, que abrangem informações sobre instituições de ensino superior públicas e privadas. Para fins de recorte metodológico, a amostra será composta pelas dez instituições de ensino superior do DF que obtiveram as melhores avaliações segundo o Índice Geral de Cursos (IGC) do Ministério da Educação (MEC) no ano de 2021.

## Justificativa
A escolha por focar apenas nas dez instituições de ensino superior mais bem avaliadas do Distrito Federal tem como objetivo garantir maior clareza e confiabilidade na análise. Essas instituições costumam ter estruturas mais organizadas e sistemas de gestão mais transparentes, o que facilita o acesso a dados mais completos e padronizados. Além disso, ao analisar faculdades com bom desempenho, buscamos entender se a evasão estudantil também ocorre em contextos considerados positivos — o que pode apontar para questões mais amplas e estruturais no ensino superior brasileiro. Essa delimitação também está relacionada ao fato de este ser um trabalho de início de graduação, o que exige um recorte de dados menor e mais viável, sem a necessidade de incluir todas as instituições do DF ou de outros estados do país.




### 2. Coleta de Dados

### Criando função para otimizara leitura dos arquivos.

In [57]:
def importando_dados(lista_caminho:list[str]):

    # Data frame criado para poder guardar a junção de todos os DataFrames. 
    juntando_dataFrames = pd.DataFrame()

    # Laço for para que ocorra a leitura de todos os arquivos csv de acordo com seus caminhos(caminho local)
    for arquivo in lista_caminho:

        # Listar todas as colunas no arquivo, afim de não baixar todo o arquivo antes de excluir as colunas rejeitadas.
        todas_colunas = pd.read_csv(arquivo, nrows=0,encoding="ISO-8859-1", sep=';').columns.tolist()

        # Definir as colunas que você quer excluir
        # Essas colunas  estão sendo tiradas por se tratar apenas de dados do distrito_Federal, sem necessidade da permanencia de informações implicitamente informadas.
        colunas_para_excluir = ['NO_REGIAO', 'CO_REGIAO','NO_UF','CO_UF','CO_MUNICIPIO','NO_MUNICIPIO','IN_CAPITAL']

        # Listar as colunas que você deseja manter
        colunas_para_manter = [coluna for coluna in todas_colunas if coluna not in colunas_para_excluir]

        # Carregar o arquivo com apenas as colunas que você quer manter
        df = pd.read_csv(arquivo, usecols=colunas_para_manter,encoding="ISO-8859-1", sep=';')

        # Utilizando do groupby para que somente as informações referente ao Distrito-Federal(DF) permaneçam.
        dados_df = df.groupby(['SG_UF'] ).get_group('DF')

        # Juntando os novos DataFrames com os anteriores. 
        juntando_dataFrames = pd.concat([dados_df,juntando_dataFrames],axis=0)

    return juntando_dataFrames

In [58]:
# Criando lista de caminhos onde os arquivos estão sendo mantidos.
caminhos_lista = [
    "D:\\analise_de_Dados\\arquivos_Brutos\\Projeto_censo_educacao_superior\\Microdados do Censo da Educação Superior 2023\\dados\\MICRODADOS_CADASTRO_CURSOS_2023.CSV",
    "D:\\analise_de_Dados\\arquivos_Brutos\\Projeto_censo_educacao_superior\\Microdados do Censo da Educação Superior 2022\\dados\\MICRODADOS_CADASTRO_CURSOS_2022.CSV",
    "D:\\analise_de_Dados\\arquivos_Brutos\\Projeto_censo_educacao_superior\\Microdados do Censo da Educação Superior 2021\\dados\\MICRODADOS_CADASTRO_CURSOS_2021.CSV",
    "D:\\analise_de_Dados\\arquivos_Brutos\\Projeto_censo_educacao_superior\\Microdados do Censo da Educação Superior 2020\\dados\\MICRODADOS_CADASTRO_CURSOS_2020.CSV",
    "D:\\analise_de_Dados\\arquivos_Brutos\\Projeto_censo_educacao_superior\\Microdados do Censo da Educação Superior 2019\\dados\\MICRODADOS_CADASTRO_CURSOS_2019.CSV",
    "D:\\analise_de_Dados\\arquivos_Brutos\\Projeto_censo_educacao_superior\\Microdados do Censo da Educação Superior 2018\\dados\\MICRODADOS_CADASTRO_CURSOS_2018.CSV",
    
]


In [59]:
# Criando o DataFrame para receber o arquivo somente com os Dados do Distrito-Federal( DF)
dados_censo_DF = importando_dados(caminhos_lista).reset_index(drop=True)

C:\Users\Thiago\AppData\Local\Temp\ipykernel_14380\288032912.py:20: DtypeWarning: Columns (4) have mixed types. Specify dtype option on import or set low_memory=False.
  df = pd.read_csv(arquivo, usecols=colunas_para_manter,encoding="ISO-8859-1", sep=';')
C:\Users\Thiago\AppData\Local\Temp\ipykernel_14380\288032912.py:23: FutureWarning: When grouping with a length-1 list-like, you will need to pass a length-1 tuple to get_group in a future version of pandas. Pass `(name,)` instead of `name` to silence this warning.
  dados_df = df.groupby(['SG_UF'] ).get_group('DF')
C:\Users\Thiago\AppData\Local\Temp\ipykernel_14380\288032912.py:20: DtypeWarning: Columns (4) have mixed types. Specify dtype option on import or set low_memory=False.
  df = pd.read_csv(arquivo, usecols=colunas_para_manter,encoding="ISO-8859-1", sep=';')
C:\Users\Thiago\AppData\Local\Temp\ipykernel_14380\288032912.py:23: FutureWarning: When grouping with a length-1 list-like, you will need to pass a length-1 tuple to get_g

In [ ]:
# Importando somente os dados sigla da universidade, para ter os nomes das mesma para melhor identificação
df_cadastro_ies = pd.read_csv("D:\\analise_de_Dados\\arquivos_Brutos\\Projeto_censo_educacao_superior\\Microdados do Censo da Educação Superior 2023\\dados\\MICRODADOS_ED_SUP_IES_2023.CSV",sep=';',encoding="ISO-8859-1",usecols=['CO_IES','SG_IES'])

In [ ]:
# Juntando Dataframe com os dados das instituições junto com suas respectivas Siglas.
dados_censo_DF_SGIS = pd.merge(dados_censo_DF,df_cadastro_ies,on='CO_IES',how='inner')

In [74]:
# Visualizando Dataframe completo.
dados_censo_DF_SGIS.head(3)

NU_ANO_CENSO SG_UF  TP_DIMENSAO  TP_ORGANIZACAO_ACADEMICA  \
0          2018    DF            1                         1   
1          2018    DF            1                         1   
2          2018    DF            1                         1   

   TP_CATEGORIA_ADMINISTRATIVA  TP_REDE  CO_IES                NO_CURSO  \
0                            1        1       2     ABI - ARTES CÊNICAS   
1                            1        1       2   ABI - ARTES PLÁSTICAS   
2                            1        1       2  ABI - CIÊNCIAS SOCIAIS   

   CO_CURSO                                  NO_CINE_ROTULO  ...  \
0   5001035                         ABI Artes e humanidades  ...   
1   5001034                         ABI Artes e humanidades  ...   
2   5001037  ABI Ciências sociais, comunicação e informação  ...   

  QT_MAT_ATIV_EXTRACURRICULAR  QT_CONC_ATIV_EXTRACURRICULAR QT_MOB_ACADEMICA  \
0                         8.0                           0.0              0.0   
1                         4.0                           0.0              0.0   
2                        24.0                           0.0              0.0   

   QT_ING_MOB_ACADEMICA QT_MAT_MOB_ACADEMICA  QT_CONC_MOB_ACADEMICA  \
0                   0.0                  0.0                    0.0   
1                   0.0                  0.0                    0.0   
2                   0.0                  0.0                    0.0   

  CO_CINE_ROTULO2  IN_COMUNITARIA  IN_CONFESSIONAL  SG_IES  
0             NaN             NaN              NaN     UNB  
1             NaN             NaN              NaN     UNB  
2             NaN             NaN              NaN     UNB  

[3 rows x 197 columns]

In [ ]:
# Visualizando todas as instituições disponíveis.
dados_censo_DF_SGIS['SG_IES'].unique()

array(['UNB', 'UNIP', 'UNICEUB', 'UCB', 'UPIS', 'UDF', 'FADM', 'IESB',
       'UNIEURO', 'Anhanguera DF', 'UNIPLAN', 'FACEF', 'UNIPROJEÇÃO',
       'ESTÁCIO BRASÍLIA', 'FAPRO', 'FACBRAS', 'FALBE', 'UNIPROCESSUS',
       nan, 'ISALBE', 'UNILS', 'FTBB', 'Unicesp', 'FACULDADE FABRAS',
       'UNIMAUÁ', 'FAC CCI', 'ICESP', 'APOGEU', 'UNICEPLAC', 'IDP-BSB',
       'FATEO', 'IFB', 'FACHORIZONTE', 'ISCP', 'FANASPS', 'FATECNA',
       'ESAS', 'FACDF', 'FPMB', 'ATAME', 'FACTI', 'ISCON', 'FACE',
       'MACKENZIE', 'UNAR', 'CLARETIANOBT', 'UNIUBE', 'UNESA', 'UMESP',
       'UNICSUL', 'Unopar', 'UB', 'FMU', 'UNISA', 'UNIFACS', 'UCDB',
       'UNICID', 'UAM', 'Univeritas UNG', 'UNISUL', 'UNICNEC', 'UNIDERP',
       'UNIGRAN', 'UNICEUMA', 'UP', 'UNICESUMAR', 'UNIFAEL',
       'ESTÁCIO RIBEIRÃO PRE', 'UNIFACCAMP', 'UNASP', 'UNIASSELVI',
       'UNIDOM - BOSCO', 'UNINTER', 'UNIGRANDE', 'AIEC / FAAB', 'EBAPE',
       'UNINASSAU', 'UNIS-MG', 'UNIFACVEST', 'SENACSP', 'UNISIGNORELLI',
       'FACAP', 'Na

| Nome da instituição                          | Sigla       | Categoria Administrativa    | Organização Acadêmica | IGC | Ano  | Link (MEC/e-MEC)                                                                                                        |
| -------------------------------------------- | ----------- | --------------------------- | --------------------- | --- | ---- | ----------------------------------------------------------------------------------------------------------------------- |
| Universidade de Brasília                     | UNB         | Pública federal             | Universidade          | 5   | 2021 | [Consulta e-MEC](https://emec.mec.gov.br/emec/consulta-cadastro/detalhes-ies/d96957f455f6405d14c6542552b0f6eb/Mg==)     |
| Instituto Federal de Brasília                | IFB         | Pública federal             | Instituto Federal     | 4   | 2021 | – (sem página e-MEC disponível)                                                                                         |
| Universidade Católica de Brasília            | UCB         | Privada sem fins lucrativos | Universidade          | 4   | 2021 | [Consulta e-MEC](https://emec.mec.gov.br/emec/consulta-cadastro/detalhes-ies/d96957f455f6405d14c6542552b0f6eb/NDAz)     |
| Centro Univ. de Brasília                     | UNICEUB     | Privada sem fins lucrativos | Centro Universitário  | 4   | 2021 | [Consulta e-MEC](https://emec.mec.gov.br/emec/consulta-cadastro/detalhes-ies/d96957f455f6405d14c6542552b0f6eb/NDAy)     |
| Centro Univ. do Distrito Federal             | UDF         | Privada com fins lucrativos | Centro Universitário  | 3   | 2022 | [Consulta e-MEC](https://emec.mec.gov.br/emec/consulta-cadastro/detalhes-ies/d96957f455f6405d14c6542552b0f6eb/NTE4)     |
| Centro Univ. IES Brasília                    | IESB        | Privada com fins lucrativos | Centro Universitário  | 3   | 2022 | [Consulta e-MEC](https://emec.mec.gov.br/emec/consulta-cadastro/detalhes-ies/d96957f455f6405d14c6542552b0f6eb/MTA2MA==) |
| Centro Univ. Planalto Central (UNICEPLAC)    | UNICEPLAC   | Privada sem fins lucrativos | Centro Universitário  | 4   | 2021 | [Consulta e-MEC](https://emec.mec.gov.br/emec/consulta-cadastro/detalhes-ies/d96957f455f6405d14c6542552b0f6eb/NTQzOQ==) |
| Centro Univ. LS (Centro Universitário LS)    | UNILS       | Privada com fins lucrativos | Centro Universitário  | 4   | 2021 | [Consulta e-MEC](https://emec.mec.gov.br/emec/consulta-cadastro/detalhes-ies/d96957f455f6405d14c6542552b0f6eb/MzM5Ng==) |
| Centro Univ. Projeção (Sobradinho)           | UNIPROJEÇÃO | Privada sem fins lucrativos | Centro Universitário  | 4   | 2021 | [Consulta e-MEC](https://emec.mec.gov.br/emec/consulta-cadastro/detalhes-ies/d96957f455f6405d14c6542552b0f6eb/MTUzOA==) |
| Instituto Brasileiro de Ensino, Desenvolvimento e Pesquisa de Brasília | IDP-BSB     | Privada    | Faculdade             | 4   | 2021 | – (sem página e-MEC disponível)                                                                                         |


In [ ]:
# SELECIONANDO AS MELHORES FACULDADES PUBLICAS E PRIVADAS DE ACORDO COM A NOTA DO MEC NO DF
censo_DF_faculdades = dados_censo_DF_SGIS[dados_censo_DF_SGIS['SG_IES'].isin(['UNB','IFB','UNILS','UCB','IESB','UNICEUB','UNICEPLAC','UDF','IDP-BSB','UNIPROJEÇÃO'])]

Durante a etapa de pré-processamento dos dados, identificamos um total de 197 variáveis disponíveis para análise. Considerando o escopo do trabalho e o objetivo de manter um foco mais direcionado, optamos por realizar uma seleção manual das colunas que poderiam contribuir de forma mais direta para a compreensão das possíveis causas de desistência nos cursos de graduação.

A seleção levou em conta a relevância das variáveis para a análise, priorizando aquelas relacionadas a características dos cursos, perfil dos alunos, formas de ingresso, turnos, modalidades, entre outros fatores potencialmente associados à evasão. Após esse processo, reduzimos o número de variáveis para 71, listadas a seguir:

['NU_ANO_CENSO','SG_IES','CO_IES','NO_CURSO','TP_ORGANIZACAO_ACADEMICA','TP_REDE','NO_CINE_ROTULO','NO_CINE_AREA_GERAL','NO_CINE_AREA_ESPECIFICA','NO_CINE_AREA_DETALHADA','TP_GRAU_ACADEMICO','IN_GRATUITO','TP_MODALIDADE_ENSINO','TP_NIVEL_ACADEMICO','QT_VG_TOTAL',
"QT_ING_ENEM","QT_ING_VESTIBULAR",'QT_INSCRITO_TOTAL_EAD','QT_VG_TOTAL_EAD','QT_VG_TOTAL_DIURNO','QT_VG_TOTAL_NOTURNO','QT_MAT', 'QT_MAT_FEM', 'QT_MAT_MASC', 'QT_MAT_DIURNO', 'QT_MAT_NOTURNO', 'QT_MAT_0_17', 'QT_MAT_18_24', 'QT_MAT_25_29', 'QT_MAT_30_34', 'QT_MAT_35_39', 'QT_MAT_40_49', 'QT_MAT_50_59', 'QT_MAT_60_MAIS', 'QT_MAT_BRANCA', 'QT_MAT_PRETA', 'QT_MAT_PARDA', 'QT_MAT_AMARELA', 'QT_MAT_INDIGENA', 'QT_MAT_CORND', 'QT_CONC', 'QT_CONC_FEM', 'QT_CONC_MASC', 'QT_CONC_DIURNO', 'QT_CONC_NOTURNO', 'QT_CONC_0_17', 'QT_CONC_18_24', 'QT_CONC_25_29', 'QT_CONC_30_34', 'QT_CONC_35_39', 'QT_CONC_40_49', 'QT_CONC_50_59', 'QT_CONC_60_MAIS', 'QT_CONC_BRANCA', 'QT_CONC_PRETA', 'QT_CONC_PARDA', 'QT_CONC_AMARELA', 'QT_CONC_INDIGENA','QT_CONC_CORND','QT_MAT_NACESTRANG','QT_CONC_NACESTRANG','QT_MAT_DEFICIENTE', 'QT_CONC_DEFICIENTE','QT_MAT_FINANC','QT_CONC_FINANC','QT_MAT_PROCESCPUBLICA', 'QT_MAT_PROCESCPRIVADA', 'QT_MAT_PROCNAOINFORMADA','QT_CONC_PROCESCPUBLICA', 'QT_CONC_PROCESCPRIVADA', 'QT_CONC_PROCNAOINFORMADA']

Essa seleção visa manter o equilíbrio entre profundidade analítica e viabilidade de tratamento dos dados, considerando as limitações de tempo e escopo de um trabalho de graduação.

        Observação: durante o tratamento e análise dos dados, é provável que esse conjunto de variáveis ainda passe por otimizações adicionais, com a exclusão de colunas redundantes ou com baixa relevância para os objetivos da pesquisa

In [70]:
# SELECIONADO COLUNAS PARA ANÁLISE.
Colunas_para_censo= ['NU_ANO_CENSO','SG_IES','CO_IES','NO_CURSO','TP_ORGANIZACAO_ACADEMICA','TP_REDE','NO_CINE_ROTULO','NO_CINE_AREA_GERAL','NO_CINE_AREA_ESPECIFICA','NO_CINE_AREA_DETALHADA','TP_GRAU_ACADEMICO','IN_GRATUITO','TP_MODALIDADE_ENSINO','TP_NIVEL_ACADEMICO','QT_VG_TOTAL',
"QT_ING_ENEM","QT_ING_VESTIBULAR",'QT_INSCRITO_TOTAL_EAD','QT_VG_TOTAL_EAD','QT_VG_TOTAL_DIURNO','QT_VG_TOTAL_NOTURNO','QT_MAT', 'QT_MAT_FEM', 'QT_MAT_MASC', 'QT_MAT_DIURNO', 'QT_MAT_NOTURNO', 'QT_MAT_0_17', 'QT_MAT_18_24', 'QT_MAT_25_29', 'QT_MAT_30_34', 'QT_MAT_35_39', 'QT_MAT_40_49', 'QT_MAT_50_59', 'QT_MAT_60_MAIS', 'QT_MAT_BRANCA', 'QT_MAT_PRETA', 'QT_MAT_PARDA', 'QT_MAT_AMARELA', 'QT_MAT_INDIGENA', 'QT_MAT_CORND', 'QT_CONC', 'QT_CONC_FEM', 'QT_CONC_MASC', 'QT_CONC_DIURNO', 'QT_CONC_NOTURNO', 'QT_CONC_0_17', 'QT_CONC_18_24', 'QT_CONC_25_29', 'QT_CONC_30_34', 'QT_CONC_35_39', 'QT_CONC_40_49', 'QT_CONC_50_59', 'QT_CONC_60_MAIS', 'QT_CONC_BRANCA', 'QT_CONC_PRETA', 'QT_CONC_PARDA', 'QT_CONC_AMARELA', 'QT_CONC_INDIGENA','QT_CONC_CORND','QT_MAT_NACESTRANG','QT_CONC_NACESTRANG','QT_MAT_DEFICIENTE', 'QT_CONC_DEFICIENTE','QT_MAT_FINANC','QT_CONC_FINANC','QT_MAT_PROCESCPUBLICA', 'QT_MAT_PROCESCPRIVADA', 'QT_MAT_PROCNAOINFORMADA','QT_CONC_PROCESCPUBLICA', 'QT_CONC_PROCESCPRIVADA', 'QT_CONC_PROCNAOINFORMADA']

In [71]:
# Quantidade de colunas escolhidas. 
len(Colunas_para_censo)

71

In [72]:
# Pegando os registros com apenas as colunas de interesse
dados_censo_DF = censo_DF_faculdades[Colunas_para_censo].reset_index(drop=True)

A amostra foi composta apenas pelas 10 faculdades do Distrito Federal consideradas as melhores pelo MEC, não representando todas as instituições da região. As faculdades selecionadas são: UNB, IFB, UNILS, UCB, IESB, UNICEUB, UNICEPLAC, UDF, IDP-BSB e UNIPROJEÇÃO.



In [ ]:
# Visualizando se as faculdades desejadas estão no DataFrame - Estão sim.
dados_censo_DF['SG_IES'].unique()

array(['UNB', 'UNICEUB', 'UCB', 'UDF', 'IESB', 'UNIPROJEÇÃO', 'UNILS',
       'UNICEPLAC', 'IDP-BSB', 'IFB'], dtype=object)

In [ ]:
dados_censo_DF.head(5)

NU_ANO_CENSO SG_IES  CO_IES                  NO_CURSO  \
0          2018    UNB       2       ABI - ARTES CÊNICAS   
1          2018    UNB       2     ABI - ARTES PLÁSTICAS   
2          2018    UNB       2    ABI - CIÊNCIAS SOCIAIS   
3          2018    UNB       2  ABI - COMUNICAÇÃO SOCIAL   
4          2018    UNB       2              ABI - DESIGN   

   TP_ORGANIZACAO_ACADEMICA  TP_REDE  \
0                         1        1   
1                         1        1   
2                         1        1   
3                         1        1   
4                         1        1   

                                   NO_CINE_ROTULO NO_CINE_AREA_GERAL  \
0                         ABI Artes e humanidades  Programas básicos   
1                         ABI Artes e humanidades  Programas básicos   
2  ABI Ciências sociais, comunicação e informação  Programas básicos   
3  ABI Ciências sociais, comunicação e informação  Programas básicos   
4                         ABI Artes e humanidades  Programas básicos   

  NO_CINE_AREA_ESPECIFICA NO_CINE_AREA_DETALHADA  ...  QT_MAT_DEFICIENTE  \
0       Programas básicos      Programas básicos  ...                0.0   
1       Programas básicos      Programas básicos  ...                0.0   
2       Programas básicos      Programas básicos  ...                0.0   
3       Programas básicos      Programas básicos  ...                0.0   
4       Programas básicos      Programas básicos  ...                0.0   

   QT_CONC_DEFICIENTE  QT_MAT_FINANC  QT_CONC_FINANC  QT_MAT_PROCESCPUBLICA  \
0                 0.0            0.0             0.0                   20.0   
1                 0.0            0.0             0.0                   20.0   
2                 0.0            0.0             0.0                   91.0   
3                 0.0            0.0             0.0                   41.0   
4                 0.0            0.0             0.0                   14.0   

   QT_MAT_PROCESCPRIVADA  QT_MAT_PROCNAOINFORMADA  QT_CONC_PROCESCPUBLICA  \
0                   25.0                      0.0                     0.0   
1                   37.0                      0.0                     0.0   
2                   74.0                      0.0                     0.0   
3                   38.0                      0.0                     0.0   
4                   25.0                      0.0                     0.0   

   QT_CONC_PROCESCPRIVADA  QT_CONC_PROCNAOINFORMADA  
0                     0.0                       0.0  
1                     0.0                       0.0  
2                     0.0                       0.0  
3                     0.0                       0.0  
4                     0.0                       0.0  

[5 rows x 71 columns]

In [ ]:
# Salvando dados somente do DF.
dados_censo_DF.to_csv("Censo_ed_sup_df.csv",sep=';',index=False)

### LIMPEZA DE DADOS

In [75]:
# Visualizando dados gerais.
dados_censo_DF.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 3173 entries, 0 to 3172
Data columns (total 71 columns):
 #   Column                    Non-Null Count  Dtype  
---  ------                    --------------  -----  
 0   NU_ANO_CENSO              3173 non-null   int64  
 1   SG_IES                    3173 non-null   object 
 2   CO_IES                    3173 non-null   int64  
 3   NO_CURSO                  3173 non-null   object 
 4   TP_ORGANIZACAO_ACADEMICA  3173 non-null   int64  
 5   TP_REDE                   3173 non-null   int64  
 6   NO_CINE_ROTULO            3173 non-null   object 
 7   NO_CINE_AREA_GERAL        3173 non-null   object 
 8   NO_CINE_AREA_ESPECIFICA   3173 non-null   object 
 9   NO_CINE_AREA_DETALHADA    3173 non-null   object 
 10  TP_GRAU_ACADEMICO         3094 non-null   float64
 11  IN_GRATUITO               3173 non-null   int64  
 12  TP_MODALIDADE_ENSINO      3173 non-null   int64  
 13  TP_NIVEL_ACADEMICO        3173 non-null   int64  
 14  QT_VG_TO